scripts/read_write_scen.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
What is this script for? To test write and read back?

If for testing, could also go in `tests`.

Some comment would be  ...